app/src/main/java/org/greenstand/android/TreeTracker/managers/UserLocationManager.kt

```diff
@@ -81,13 +89,16 @@ class UserLocationManager(private val locationManager: LocationManager,
         } ?: false
     }
 
-    private fun hasLocationPermissions() : Boolean {
-        return ContextCompat.checkSelfPermission(context,
-                                                 android.Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_GRANTED
-                || ContextCompat.checkSelfPermission(context,
-                                                     android.Manifest.permission.ACCESS_COARSE_LOCATION) == PackageManager.PERMISSION_GRANTED
+    private fun hasLocationPermissions(): Boolean {
+        return ContextCompat.checkSelfPermission(
```

Comment:
I know this is probably for the klint, but I think we should format this ourself. This is very hard to read